Formula/spirv-cross.rb

```diff
@@ -1,10 +1,14 @@
 class SpirvCross < Formula
   desc "Performing reflection and disassembling SPIR-V"
   homepage "https://github.com/KhronosGroup/SPIRV-Cross"
-  url "https://github.com/KhronosGroup/SPIRV-Cross/archive/2021-01-15.tar.gz"
-  version "2021-01-15"
-  sha256 "d700863b548cbc7f27a678cee305f561669a126eb2cc11d36a7023dfc462b9c4"
-  license "Apache-2.0"
+  url "https://github.com/KhronosGroup/SPIRV-Cross/archive/refs/tags/sdk-1.3.243.0.tar.gz"
```

Comment:
is there any reference for such version scheme change?

Reply:
Not beyond the lack of any "Releases" after January 2021.  The tags are named according to the Vulkan SDK version they're included with.